sbol_utilities/default_components.py

```diff
@@ -0,0 +1,148 @@
+from typing import List, Tuple
```

Comment:
Could this module be combined with the existing `sbol_utilities.component` module?

Reply:
Yes, I will add the functions to `sbol_utilities.component` module